walkthroughs/howto-k8s-timeout-policy/v1beta2/manifest.yaml.template

```diff
@@ -0,0 +1,259 @@
+          perRequest:
+            unit: s
+            value: 60
+  serviceDiscovery:
+    awsCloudMap:
+      namespaceName: ${CLOUDMAP_NAMESPACE}
+      serviceName: colorapp
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: colorapp-red
+  namespace: ${APP_NAMESPACE}
+spec:
+  podSelector:
+    matchLabels:
+      app: colorapp
+      version: red
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+      timeout:
+        http:
+          perRequest:
+            unit: s
+            value: 60
+  serviceDiscovery:
+    awsCloudMap:
+      namespaceName: ${CLOUDMAP_NAMESPACE}
+      serviceName: colorapp
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualService
+metadata:
+  name: colorapp
+  namespace: ${APP_NAMESPACE}
+spec:
+  awsName: colorapp.${APP_NAMESPACE}.pvt.aws.local
+  provider:
+    virtualRouter:
+      virtualRouterRef:
+        name: colorapp-router
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualRouter
+metadata:
+  namespace: ${APP_NAMESPACE}
+  name: colorapp-router
+spec:
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+  routes:
+    - name: color-route
+      httpRoute:
+        match:
+          prefix: /
+        action:
+          weightedTargets:
+            - virtualNodeRef:
+                name: colorapp-red
+              weight: 1
+            - virtualNodeRef:
+                name: colorapp-blue
+              weight: 1
+        timeout:
+          perRequest:
+            unit: s
+            value: 60
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualService
+metadata:
+  name: front
+  namespace: ${APP_NAMESPACE}
+spec:
+  awsName: front.${APP_NAMESPACE}.pvt.aws.local
+  provider:
+    virtualRouter:
+      virtualRouterRef:
+        name: front-router
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualRouter
+metadata:
+  namespace: ${APP_NAMESPACE}
+  name: front-router
+spec:
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+  routes:
+    - name: color-route
+      httpRoute:
+        match:
+          prefix: /color
+        action:
+          weightedTargets:
+            - virtualNodeRef:
+                name: front
+              weight: 1
+        timeout:
+          perRequest:
+            unit: s
+            value: 60
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: front
+  namespace: ${APP_NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: front
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: front
+        version: v1
+    spec:
+      containers:
+        - name: front
+          image: ${FRONT_APP_IMAGE}
+          ports:
+            - containerPort: 8080
+          env:
+            - name: "PORT"
+              value: "8080"
+            - name: "COLOR_HOST"
+              value: "colorapp.${CLOUDMAP_NAMESPACE}:8080"
+          securityContext:
+            privileged: true
```

Comment:
What's the reason for this to be a privileged container?

Reply:
Not needed. Added it to modify iptables rules while debugging a traffic issue. Will remove it.